### ISSUE TITLE
Card: There is no separate visible focus provided for the card. Due to which user is unable to navigate to the other cards present within the screen.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3814]
Description: A mechanism is unavailable to bypass blocks of content that are repeated on multiple Web pages.
Here, there is no separate visible focus provided for the card. Due to which user is unable to navigate to the other cards present within the screen.

Expected: Application should provide skip option for large and complex UI elements.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98772689-4d995000-240d-11eb-9478-01d1de78ca61.png)

**Comment by DeepakSap14** [comment]
This has been fixed in PR https://github.com/SAP/fundamental-ngx/pull/4330.
please verify with latest master.

![Screenshot 2021-01-29 at 2.51.02 PM.png](https://images.zenhubusercontent.com/5f4ccf79136fcb712e6260b3/76858246-433b-4c4b-8dbf-088945e53ffa)

**Comment by puru-hk** [comment]
Actually for card, visible focus is provided but for navigation from one card to another, there is no shortcut key combination or any other alternative provided.

**Comment by DeepakSap14** [comment]
Do we have any such requirement from fiori?
i don't see it.

Even i saw grid list component does not have this kind of implementation.

**Comment by puru-hk** [comment]
It's an accessibility requirement. If it is not working in grid list I will raise an issue.

**Comment by DeepakSap14** [comment]
checked Angular material implementation as well. They also do not provide such accessibility on page level.
It is a layout, which spreads on whole page. so tab order should be followed. which will access the card and its content. it will not skip any card/area.
Do you have some document, which i can follow?

**Comment by puru-hk** [comment]
Tested it is working as expected
